april_vision/cli/live.py

```diff
@@ -82,7 +84,7 @@ def main(args: argparse.Namespace):
                         text_colour=(255, 191, 0),  # deep sky blue
                     )
             except RuntimeError:
-                pass
+                raise
```

Comment:
This is a change of behaviour.
```suggestion
                pass
```